src/error.rs

```diff
@@ -66,7 +60,7 @@ where
 }
 
 impl<'jvm> Error<Local<'jvm, Throwable>> {
-    pub fn into_global(self, jvm: &mut Jvm<'jvm>) -> Error<Global<Throwable>> {
```

Comment:
should this still be called `into_global`? It seems we're mostly erradicating the word from the codebase